requirements.txt

```diff
@@ -0,0 +1,12 @@
+aspy.yaml==0.2.2
```

Comment:
You may check the way it works in @pavlenk0's [repo](https://github.com/kpi-web-guild/django-girls-blog-pavlenk0/blob/master/test_requirements.txt)